src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -324,22 +383,21 @@ public static class PublishedContentExtensions
         /// <returns>The <see cref="object"/> representing the Umbraco value.</returns>
         /// <param name="instance">The instance to assign the value to.</param>
         private static object GetRawValue(
-           IPublishedContent content,
-            Type type,
+            IPublishedContent content,
             CultureInfo culture,
             PropertyInfo propertyInfo,
-            ref object instance)
+            object instance)
```

Comment:
In this method it shouldn't make a difference, but in the SetTypedValue method I'm not too sure. propertyInfo.SetValue gets called in that method meaning it modifies the instance object. If not passed in by ref then the calling code wouldn't get the updated changes. I'd have to look through the codebase to see what calls this method and whether it expects the instance object to be updated or not.